pycc/lccwfn.py

```diff
@@ -0,0 +1,476 @@
+
+            #ldiis.add_error_vector(self.t1_ii,self.t2_ij)
+            #if niter >= start_diis:
+                #self.t1_ii, self.t2_ij = ldiis.extrapolate(self.t1_ii, self.t2_ij)
+
+    def local_residuals(self, t1_ii, t2_ij):
+        """
+        Constructing the two- and four-index intermediates
+        Then evaluating the singles and doubles residuals, storing them in a list of length occ (single) and length occ*occ (doubles)
+        Naming scheme same as integrals where _ij is attach as a suffix to the intermediate name 
+        ... special naming scheme for those involving two different pair space ij and im -> _ijm
+
+        To do
+        ------
+        There are some arguments that isn't really being used and will be updated once things are good to go
+        Listed here are intermediates with corresponding arguments that aren't needed since it requires "on the fly" generation
+        Fae_ij Lovvv_ij
+        Fme_ij (Fme_im) Loovv_ij
+        Wmbej_ijim ERIovvv_ij, ERIoovv_ij, Loovv_ij
+        Wmbje_ijim (Wmbie_ijmj) ERIovvv_ij, ERIoovv_ij
+        r1_ii Lovvo_ij, ERIovvo_ij
+        r2_ij ERIovvo_ij, ERIovov_ij, ERIvvvo_ij
+        """
+        o = self.o
+        v = self.v
+        F = self.H.F
+        L = self.H.L
+        ERI = self.H.ERI
+
+        Fae_ij = []
+        Fme_ij = []
+
+        Wmbej_ijim = []
+        Wmbje_ijim = []
+        Wmbie_ijmj = []
+        Zmbij_ij = []
+
+        r1_ii = []
+        r2_ij = []
+
+        Fae_ij = self.build_lFae(Fae_ij, self.Local.Fvv_ij, self.Local.Fov_ij, 
+        self.Local.Lovvv_ij, self.Local.Loovv_ij, t1_ii, t2_ij)
+        lFmi = self.build_lFmi(o, F, self.Local.Fov_ij, self.Local.Looov_ij, self.Local.Loovv_ij, t1_ii, t2_ij)
+        Fme_ij = self.build_lFme(Fme_ij, self.Local.Fov_ij, self.Local.Loovv_ij, t1_ii)
+        lWmnij = self.build_lWmnij(o, ERI, self.Local.ERIooov_ij, self.Local.ERIoovo_ij, 
+        self.Local.ERIoovv_ij, t1_ii, t2_ij)
+        Zmbij = self.build_lZmbij(Zmbij_ij, self.Local.ERIovvv_ij, t1_ii, t2_ij)
+        Wmbej_ijim = self.build_lWmbej(Wmbej_ijim, self.Local.ERIoovv_ij, self.Local.ERIovvo_ij, 
+        self.Local.ERIovvv_ij,self.Local.ERIoovo_ij, self.Local.Loovv_ij, t1_ii, t2_ij)
+        Wmbje_ijim, Wmbie_ijmj = self.build_lWmbje(Wmbje_ijim, Wmbie_ijmj, self.Local.ERIovov_ij, 
+        self.Local.ERIovvv_ij,self.Local.ERIoovv_ij, self.Local.ERIooov_ij, t1_ii, t2_ij)
+
+        r1_ii = self.lr_T1(r1_ii, self.Local.Fov_ij , self.Local.ERIovvv_ij, self.Local.Lovvo_ij, self.Local.Loovo_ij, 
+        t1_ii, t2_ij,Fae_ij, Fme_ij, lFmi)
+        r2_ij = self.lr_T2(r2_ij, self.Local.ERIoovv_ij, self.Local.ERIvvvv_ij, self.Local.ERIovvo_ij, self.Local.ERIovoo_ij, 
+        self.Local.ERIvvvo_ij,self.Local.ERIovov_ij, t1_ii, t2_ij, Fae_ij ,lFmi,Fme_ij, lWmnij, Zmbij_ij, Wmbej_ijim, Wmbje_ijim, Wmbie_ijmj)
+
+        return r1_ii, r2_ij    
+
+    def build_lFae(self, Fae_ij, Fvv_ij,Fov_ij, Lovvv_ij, Loovv_ij, t1_ii, t2_ij):
```

Comment:
Of these arguments, it looks like only `Fae_ij` `Fvv_ij`, and `t2_ij` are used.  Why not eliminate the unused arguments?